src/control/server/mgmt_system.go

```diff
@@ -1138,7 +1145,56 @@ func (svc *mgmtSvc) SystemSetProp(ctx context.Context, req *mgmtpb.SystemSetProp
 		return nil, err
 	}
 
-	return &mgmtpb.DaosResp{}, nil
+	// for any system property that has a pool property, update all pools
+	var poolSysProps []*daos.PoolProperty
+	for k := range req.GetProperties() {
+		p, ok := svc.systemProps.Get(k)
+		if !ok {
+			return nil, errors.Errorf("unknown property %q", k)
+		}
+		if pp, ok := sp2pp(p); ok {
+			poolSysProps = append(poolSysProps, pp)
+		}
+	}
+
+	resp = new(mgmtpb.DaosResp)
+	if len(poolSysProps) == 0 {
+		return
+	}
+
+	pspr := &mgmtpb.PoolSetPropReq{
+		Sys:        req.Sys,
+		Properties: make([]*mgmtpb.PoolProperty, len(poolSysProps)),
+	}
+	for i, p := range poolSysProps {
+		pspr.Properties[i] = &mgmtpb.PoolProperty{
+			Number: uint32(p.Number),
+		}
+		if nv, err := p.Value.GetNumber(); err == nil {
+			pspr.Properties[i].SetValueNumber(nv)
+		} else {
+			pspr.Properties[i].SetValueString(p.Value.String())
+		}
+	}
+
+	pools, err := svc.sysdb.PoolServiceList(false)
+	if err != nil {
+		return nil, err
+	}
+	for _, ps := range pools {
```

Comment:
Could the `PoolSetSystemProp` function below share some of this logic? Seems like they're basically doing the same thing.

Reply:
Actually PoolSetSystemProp isn't used anymore. Deleted that function.